### DISCUSSION TITLE
[Feature] add people on photo

### DISCUSSION THREAD

**Discussion by nodiaque** [immich-app/immich#14381] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

It would be nice if we could add people in photo information that isn't detected. I have a picture where some person aren't found by the face detection thus they don't appear when I go into person and click them. Having the ability to say this person is in that photo even without a face recognized would be great. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7301
